app/constants.go

```diff
@@ -30,8 +30,11 @@ var (
 		"ibc/C8A74ABBE2AF892E15680D916A7C22130585CE5704F9B17A10F184A90D53BECA", // ATOM
 		"ibc/A6E3AF63B3C906416A9AF7A556C59EA4BD50E617EFFE6299B99700CCB780E444", // pSTAKE
 		"ibc/646315E3B0461F5FA4C5C8968A88FC45D4D5D04A45B98F1B8294DD82F386DD85", // OSMO
+		"ibc/23DC3FF0E4CBB53A1915E4C62507CB7796956E84C68CA49707787CB8BDE90A1E", // DYDX
+		"ibc/B3792E4A62DF4A934EF2DF5968556DB56F5776ED25BDE11188A4F58A7DD406F0", // USDC
 		"stk/uatom", // stkATOM
 		"stk/uosmo", // stkOSMO
+		"stk/adydx", // stkDYDX
 	}
```

Comment:
@xlab cherry-picked this changes from v10.4.1-fix.. can you once run diff check from v10.4.1-fix to main branch? 
i think only this should be missing..